cogs5e/models/sheet/base.py

```diff
@@ -28,27 +28,40 @@ def to_dict(self):
     @classmethod
     def default(cls):
         return cls(0, 10, 10, 10, 10, 10, 10)
+    
+    @classmethod
+    def custstats(cls,cust,origstats):
```

Comment:
nit(code-style): variable name verbosity

Reply:
nit: missing documentation - this method appears to construct a new BaseStats given a dict (of what type?) and another BaseStats instance

This feels like it would be implemented better with the addition of a `__copy__` method and a `.update(other: BaseStats)` method, or even better, construct a new BaseStats instance directly where needed (possibly using `from_dict` and zipping the stat name constants).